datahub-web-react/src/app/ingest/ManageIngestionPage.tsx

```diff
@@ -49,6 +51,13 @@ export const ManageIngestionPage = () => {
      * Determines which view should be visible: ingestion sources or secrets.
      */
     const [selectedTab, setSelectedTab] = useState<TabType>(TabType.Sources);
+    const me = useUserContext();
+    const { config } = useAppConfig();
+    const isIngestionEnabled = config?.managedIngestionConfig.enabled;
+    const showIngestionTab =
+        isIngestionEnabled && me && me.platformPrivileges?.manageIngestion;
+    const showSecretsTab =
+        isIngestionEnabled && me && me.platformPrivileges?.manageSecrets;
```

Comment:
since users can now get here if they can manage ingestion or secrets, let's set the default `selectedTab` above on line 53 to `TabType.Secrets` if they can only view secrets and not ingestion

Reply:
++

Reply:
Makes sense. Will do, test and then mark this as ready for review.